web/html/src/components/dialog/ActionConfirm.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+import { DangerDialog } from "./DangerDialog";
+
+type Props = {
+  id: string;
+  type: string;
+  name: string;
+  itemName: string;
+  icon: string;
+  selected: any[];
+  // TODO: This should be renamed `onConfirm` or something similar
```

Comment:
Feel free to do the rename, `onConfirm` makes sense to me as it is used only in that case IIUC.

Reply:
Thanks, I went ahead and changed all the occurrences.